src/tUltimateLI.pas

```diff
@@ -604,6 +607,25 @@ procedure TuLI.ParseDeviceMsg(deviceAddr: Byte; var msg: TBuffer);
               Self.SendLocoData(msg.data[0], Self.LokAddrDecode(msg.data[3],
                 msg.data[4]));
             end;
+          07:
+            begin
+              Self.WriteLog(tllCommands, 'GET: locomotive Funktionsstatus F0-F12 request (>=3.6)');
+              Self.SendLocoFuncMomentary(msg.data[0], Self.LokAddrDecode(msg.data[3],
+                msg.data[4]));
+            end;
+          08:
+            begin
+              Self.WriteLog(tllCommands, 'GET: locomotive Funktionsstatus F13-F28 request (>=3.6)');
+              Self.SendLocoFunc13Momentary(msg.data[0], Self.LokAddrDecode(msg.data[3],
+                msg.data[4]));
+            end;
+          09:
+            begin
+              Self.WriteLog(tllCommands, 'GET: locomotive Funktionszustand F13-F28 request (>=3.6)');
+              Self.SendLocoFunc13(msg.data[0], Self.LokAddrDecode(msg.data[3],
+                msg.data[4]));
+            end;
+
```

Comment:
Přidány zprávy dle specifikace 3.6 - vyčteno z dokumentu "XpressNET version 4.0"
V logu použíty originální nazvyzpráv, nepodařilo se mi to přeložit správně.